### DISCUSSION TITLE
Custom SSL - Must Docker be restarted everytime I update the contents of the .crt and .key custom SSL files?

### DISCUSSION THREAD

**Discussion by BornKillaz** [AzuraCast/AzuraCast#7096] [Q&A]
Hi!

I have a question regarding the management of SSL certificates in a Docker setup. I'm currently using AzuraCast and have my SSL certificates automatically renewing every three months through a custom cron job. 

Here is the relevant section from my docker-compose.override.yml:
services:
```
  web:
    volumes:
      - /var/azuracast/fullchain.crt:/var/azuracast/acme/ssl.crt:ro
      - /var/azuracast/key.key:/var/azuracast/acme/ssl.key:ro
```
     

**The filenames of the certificates (fullchain.crt and key.key) do not change, only their contents are updated upon renewal.**

Do I need to restart the Docker containers every time these SSL certificates are updated, or is there a mechanism within AzuraCast or Docker that detects and applies these updates without requiring a container restart? 

Thank you for your assistance.

**Comment by BusterNeece** [comment]
I'm realizing now I didn't get back to posting this here, but a few months ago, a new CLI command was created that you can execute (i.e. inside a cron job) that'll reload all services inside AzuraCast that depend on an SSL certificate:

```bash
cd /var/azuracast
bash docker.sh cli azuracast:acme:reload
```
